### PR TITLE
Add '*.wproj' Test

### PR DESCRIPTION
**Pull request by DavidMansolino**
This PR:
  - [x] Add a test to 'test_project_structure.py' to make sure every world file has an associated proj file.
  - [x] Improve the individual test to let them work as standalone executables.

### CODE REVIEW

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
:+1:

File: tests/sources/test_project_structure.py [hunk 1]
```diff
@@ -51,6 +53,17 @@ def test_world_directories(self):
                     msg='This world is nested because an upper "worlds" directory exists: "%s"' % world
                 )
 
+    def test_wbproj(self):
+        """Test that each world has an associated '.wproj' file."""
```
Comment (omichel) [inline tests/sources/test_project_structure.py]:
```suggestion
        """Test that each world has an associated '.wbproj' file."""
```

**Comment by DavidMansolino** [comment]
:+1:

**Pull request merged**